[package] qt/6.8.3: Undefined symbols for `CAMetalLayer` when linking against static Qt Gui

### Description

When linking against a statically-linked `qt`, I get the following error:
```
Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$_CAMetalLayer", referenced from:
      _OBJC_CLASS_$_QMetalLayer in libQt6Gui.a[197](qmetallayer.mm.o)
  "_OBJC_METACLASS_$_CAMetalLayer", referenced from:
      _OBJC_METACLASS_$_QMetalLayer in libQt6Gui.a[197](qmetallayer.mm.o)
ld: symbol(s) not found for architecture x86_64
```

It seems that `CAMetalLayer` from `QuartzCore` is missing, which indicates a missing `-framework QuartzCore`. It seems to me that this should be added in `package_info()` for [`macOS`, `iOS`, and `visionOS`](https://github.com/qt/qtbase/blob/6.8.3/src/gui/configure.cmake#L841-L844) when `gui` is enabled and `metal` is not disabled, see https://github.com/qt/qtbase/blob/6.8.3/src/gui/CMakeLists.txt#L434-L439.

See PR at #28914.

### Package and Environment Details

* Package Name/Version: **qt/6.8.3** (I don't think 6.7.3 is affected)
* Compiler+version: **Apple clang version 17.0.0**

```
version: 2.21.0
conan_path: /usr/local/bin/conan
python
  version: 3.14.0
  sys_version: 3.14.0 (main, Oct  7 2025, 09:34:52) [Clang 17.0.0 (clang-1700.0.13.3)]
  sys_executable: /usr/local/Cellar/conan/2.21.0/libexec/bin/python
  is_frozen: False
  architecture: x86_64
system
  version: Darwin Kernel Version 24.5.0: Tue Apr 22 19:53:26 PDT 2025; root:xnu-11417.121.6~2/RELEASE_X86_64
  platform: macOS-15.5-x86_64-i386-64bit-Mach-O
  system: Darwin
  release: 24.5.0
  cpu: i386
```

### Conan profile

```
[settings]
arch=x86_64
build_type=Release
compiler=apple-clang
compiler.cppstd=20
compiler.libcxx=libc++
compiler.version=17
os=Macos
os.version=13.3

[options]
openssl/*:openssldir=/etc/ssl/

[conf]
tools.build:compiler_executables={'c': 'clang', 'cpp': 'clang++'}
tools.cmake.cmaketoolchain:generator=Ninja
# We only have Xcode command line tools installed, but this always worked fine with Qt 6.7
qt/*:tools.cmake.cmaketoolchain:extra_variables={'QT_NO_XCODE_MIN_VERSION_CHECK': 'ON'}
```

### Steps to reproduce

- Install `qt/6.8.3` as static library (`shared=False`)
- Link against an executable using Qt Gui

### Logs

<details><summary>Click to expand log</summary>

```
======== Computing dependency graph ========
Graph root
    conanfile.py (pep/None): /Users/gitlab/builds/SF_xY11J8/0/pep/core/conanfile.py
Requirements
    boost/1.89.0#010f59feedfd171b15f467b42f723d13 - Cache
    bzip2/1.0.8#c470882369c2d95c5c77e970c0c7e321 - Cache
    civetweb/1.16#f4a42c8aa0bc0fe4bbe2a2270af0cea2 - Cache
    double-conversion/3.3.0#6ad026cf92c316c476919f7f152072d7 - Cache
    inja/3.5.0#ddfa08eb3c763bfe58d4c2bffe88f828 - Cache
    libarchive/3.8.1#ffee18995c706e02bf96e7a2f7042e0d - Cache
    libpng/1.6.50#c96b3b9fa67d44545d6583bb0c348904 - Cache
    nlohmann_json/3.12.0#2d634ab0ec8d9f56353e5ccef6d6612c - Cache
    opengl/system#7c02ea6ef926fd04844af53622a30541 - Cache
    openssl/3.6.0#89e8af1d4a21afcac0557079d23d8890 - Cache
    pcre2/10.42#9a35f5089feb875ec61a38eca364ce77 - Cache
    protobuf/3.21.12#44ee56c0a6eea0c19aeeaca680370b88 - Cache
    qt/6.8.3#f6ec3d69031717def8d69ad53bcf8558 - Cache
    sqlite3/3.50.4#fc0579bfa505c439d8abaf2af68e2697 - Cache
    sqlite_orm/1.9.1#b0fc2944b635b1a83bba7ae3c61315bb - Cache
    xxhash/0.8.3#681d36a0a6111fc56e5e45ea182c19cc - Cache
    zlib/1.3.1#b8bc2603263cf7eccbd6e17e66b0ed76 - Cache
Build requirements
    b2/5.3.3#107c15377719889654eb9a162a673975 - Cache
    bzip2/1.0.8#c470882369c2d95c5c77e970c0c7e321 - Cache
    cmake/3.31.9#f9b9bb4bdfa37937619a33e9218703a6 - Cache
    cmake/4.1.2#6f844b9ec4e0e29d4f1232f9c5039901 - Cache
    double-conversion/3.3.0#6ad026cf92c316c476919f7f152072d7 - Cache
    libpng/1.6.50#c96b3b9fa67d44545d6583bb0c348904 - Cache
    meson/1.9.1#abbc783cd297bedce14581b4aec060b8 - Cache
    ninja/1.13.1#294f8721dbcde145674f7ba44994700e - Cache
    opengl/system#7c02ea6ef926fd04844af53622a30541 - Cache
    openssl/3.6.0#89e8af1d4a21afcac0557079d23d8890 - Cache
    pcre2/10.42#9a35f5089feb875ec61a38eca364ce77 - Cache
    pkgconf/2.5.1#93c2051284cba1279494a43a4fcfeae2 - Cache
    protobuf/3.21.12#44ee56c0a6eea0c19aeeaca680370b88 - Cache
    qt/6.8.3#f6ec3d69031717def8d69ad53bcf8558 - Cache
    zlib/1.3.1#b8bc2603263cf7eccbd6e17e66b0ed76 - Cache
Options conflicts
    openssl/3.6.0:no_legacy=False (current value)
        pep/None->no_legacy=True
    openssl/3.6.0:no_md4=False (current value)
        pep/None->no_md4=True
    openssl/3.6.0:no_rc2=False (current value)
        pep/None->no_rc2=True
    openssl/3.6.0:no_rc4=False (current value)
        pep/None->no_rc4=True
    openssl/3.6.0:no_ssl3=False (current value)
        pep/None->no_ssl3=True
    It is recommended to define options values in profiles, not in recipes
WARN: risk: There are options conflicts in the dependency graph
======== Computing necessary packages ========
Connecting to remote 'conancenter' anonymously
qt/6.8.3: Main binary package '92b716dd6a5affd19225a692e1b7945296daba4f' missing
qt/6.8.3: Checking 7 compatible configurations
qt/6.8.3: Compatible configurations not found in cache, checking servers
qt/6.8.3: '4dedc0cb7be149fed511ce76554b71d905fd87ab': compiler.cppstd=17
qt/6.8.3: 'da027a2cec03feec766384cccfd97a73fe52975f': compiler.cppstd=gnu17
qt/6.8.3: '9e111bff491905cf17b0d2133b0a6e4ae738acf0': compiler.cppstd=gnu20
qt/6.8.3: '4822be860592e015958136bb60f03c91eabd60b9': compiler.cppstd=23
qt/6.8.3: 'b83b6541519d50c37d49aa229d328aeafc4d1c6d': compiler.cppstd=gnu23
qt/6.8.3: '630eba11b285e6eba654a5902431440fc2f91d58': compiler.cppstd=26
qt/6.8.3: 'c8a42bdb0e00d3e8b987af382691849568da6a3b': compiler.cppstd=gnu26
Requirements
    boost/1.89.0#010f59feedfd171b15f467b42f723d13:68395a36a8de653c9fe1bc96c91a8e80608f92a4#10a200988793b2b1b452c96f2ac534d4 - Cache
    bzip2/1.0.8#c470882369c2d95c5c77e970c0c7e321:92524e955a78de607560f3d119972257761adfb7#39da8ec681f872cbca7d5e2aede62ba4 - Cache
    civetweb/1.16#f4a42c8aa0bc0fe4bbe2a2270af0cea2:eff40ca3366de2ee5d51ee154e0ef1f5030df155#181399bdae89ebd421f36490cec007f2 - Cache
    double-conversion/3.3.0#6ad026cf92c316c476919f7f152072d7:6952ca76c973e6ae7794b3ebd3b416a44ab1e828#8a6b6e50758e7bb216c95cb03f9d0188 - Cache
    inja/3.5.0#ddfa08eb3c763bfe58d4c2bffe88f828:da39a3ee5e6b4b0d3255bfef95601890afd80709#87943b6ff8a34829abee9f0977c5e9de - Cache
    libarchive/3.8.1#ffee18995c706e02bf96e7a2f7042e0d:d8887746bdf5291e042256563765df990496dea7#8dd26dde3a9711aadacaeb93ebe70c82 - Cache
    libpng/1.6.50#c96b3b9fa67d44545d6583bb0c348904:667418d03656171baefdf04cb737747680bcf542#feeb3c4660e1c050eb23b0440830ac3c - Cache
    nlohmann_json/3.12.0#2d634ab0ec8d9f56353e5ccef6d6612c:da39a3ee5e6b4b0d3255bfef95601890afd80709#2ea4b814fc9da53f7cdf5b7d68052688 - Cache
    opengl/system#7c02ea6ef926fd04844af53622a30541:da39a3ee5e6b4b0d3255bfef95601890afd80709#0ba8627bd47edc3a501e8f0eb9a79e5e - Cache
    openssl/3.6.0#89e8af1d4a21afcac0557079d23d8890:61b6a0a869ab58b4b70490ed1571130a31cf25c4#67bd9e5c4c00819bbdc4c614a446d311 - Cache
    pcre2/10.42#9a35f5089feb875ec61a38eca364ce77:a962a689a4e58bc1d6833d11cd8e0b44dbba1f04#2eba14acd4d25cd5049e5450f23619cd - Cache
    protobuf/3.21.12#44ee56c0a6eea0c19aeeaca680370b88:cd87b333462d2c5da1f04c3515164f0443ece730#5ed0a90280d4f12b1f71772da06b969a - Cache
    qt/6.8.3#f6ec3d69031717def8d69ad53bcf8558:92b716dd6a5affd19225a692e1b7945296daba4f - Build
    sqlite3/3.50.4#fc0579bfa505c439d8abaf2af68e2697:9c07893663c9e7bbcd93d057c9303f883db8f5c5#c8beaf657bc7d165fc53df6e7da94171 - Cache
    sqlite_orm/1.9.1#b0fc2944b635b1a83bba7ae3c61315bb:da39a3ee5e6b4b0d3255bfef95601890afd80709#3b36ceec280908d4cdb00216f3d787b5 - Cache
    xxhash/0.8.3#681d36a0a6111fc56e5e45ea182c19cc:21cc1cc4f7d794b61bfaf33506200ccad24fa8e2#668cc81927b58995cdca46d68cc49293 - Cache
    zlib/1.3.1#b8bc2603263cf7eccbd6e17e66b0ed76:f01a4b71394e0d2e53cd3251a0fa233f9809fa48#4c9e7f1029a0e4c81ccf5583cb54348e - Cache
Build requirements
    bzip2/1.0.8#c470882369c2d95c5c77e970c0c7e321:92524e955a78de607560f3d119972257761adfb7#39da8ec681f872cbca7d5e2aede62ba4 - Cache
    cmake/3.31.9#f9b9bb4bdfa37937619a33e9218703a6:707cedb4ebd4a01204da47eb5bc12652dc7891de#c8198c4546b97b49156ae1ebd0693ba3 - Cache
    double-conversion/3.3.0#6ad026cf92c316c476919f7f152072d7:6952ca76c973e6ae7794b3ebd3b416a44ab1e828#8a6b6e50758e7bb216c95cb03f9d0188 - Cache
    libpng/1.6.50#c96b3b9fa67d44545d6583bb0c348904:667418d03656171baefdf04cb737747680bcf542#feeb3c4660e1c050eb23b0440830ac3c - Cache
    ninja/1.13.1#294f8721dbcde145674f7ba44994700e:d1549e559cea9580f2b39bb54a7400c745bf7478#dbc27b82dc15e1cd27ecf40d1b5e40e9 - Cache
    opengl/system#7c02ea6ef926fd04844af53622a30541:da39a3ee5e6b4b0d3255bfef95601890afd80709#0ba8627bd47edc3a501e8f0eb9a79e5e - Cache
    openssl/3.6.0#89e8af1d4a21afcac0557079d23d8890:61b6a0a869ab58b4b70490ed1571130a31cf25c4#67bd9e5c4c00819bbdc4c614a446d311 - Cache
    pcre2/10.42#9a35f5089feb875ec61a38eca364ce77:a962a689a4e58bc1d6833d11cd8e0b44dbba1f04#2eba14acd4d25cd5049e5450f23619cd - Cache
    pkgconf/2.5.1#93c2051284cba1279494a43a4fcfeae2:6161d3186e90c20057254458839666f1ac720111#540bc809c006daf227496f2cfa3ad2cf - Cache
    protobuf/3.21.12#44ee56c0a6eea0c19aeeaca680370b88:cd87b333462d2c5da1f04c3515164f0443ece730#5ed0a90280d4f12b1f71772da06b969a - Cache
    qt/6.8.3#f6ec3d69031717def8d69ad53bcf8558:92b716dd6a5affd19225a692e1b7945296daba4f - Build
    zlib/1.3.1#b8bc2603263cf7eccbd6e17e66b0ed76:f01a4b71394e0d2e53cd3251a0fa233f9809fa48#4c9e7f1029a0e4c81ccf5583cb54348e - Cache
Skipped binaries
    b2/5.3.3, cmake/4.1.2, meson/1.9.1
======== Installing packages ========
bzip2/1.0.8: Already installed! (1 of 20)
cmake/3.31.9: Already installed! (2 of 20)
double-conversion/3.3.0: Already installed! (3 of 20)
ninja/1.13.1: Already installed! (4 of 20)
nlohmann_json/3.12.0: Already installed! (5 of 20)
opengl/system: Already installed! (6 of 20)
sqlite3/3.50.4: Already installed! (7 of 20)
xxhash/0.8.3: Already installed! (8 of 20)
zlib/1.3.1: Already installed! (9 of 20)
boost/1.89.0: Already installed! (10 of 20)
libarchive/3.8.1: Already installed! (11 of 20)
pkgconf/2.5.1: Already installed! (12 of 20)
inja/3.5.0: Already installed! (13 of 20)
libpng/1.6.50: Already installed! (14 of 20)
openssl/3.6.0: Already installed! (15 of 20)
pcre2/10.42: Already installed! (16 of 20)
protobuf/3.21.12: Already installed! (17 of 20)
sqlite_orm/1.9.1: Already installed! (18 of 20)
civetweb/1.16: Already installed! (19 of 20)
qt/6.8.3: Sources downloaded from 'conancenter'
qt/6.8.3: Calling source() in /Users/gitlab/.conan2/p/qt38d4ea8bc0ce1/s/src
qt/6.8.3: Downloading 994.8MB qt-everywhere-src-6.8.3.tar.xz
qt/6.8.3: Uncompressing qt-everywhere-src-6.8.3.tar.xz to /Users/gitlab/.conan2/p/qt38d4ea8bc0ce1/s/src
qt/6.8.3: Apply patch (file): patches/remove-linkage-to-agl-framework.patch
qt/6.8.3: Apply patch (file): patches/macos-link-platform-plugin-to-foundation.patch
qt/6.8.3: Apply patch (bugfix): Workaround for too long .rps file name
qt/6.8.3: Apply patch (file): patches/fix_webengine_on_VS2022_v17.14_6.8.x.diff
-------- Installing package qt/6.8.3 (20 of 20) --------
qt/6.8.3: Building from source
qt/6.8.3: Package qt/6.8.3:92b716dd6a5affd19225a692e1b7945296daba4f
qt/6.8.3: settings: os=Macos os.version=13.3 arch=x86_64 compiler=apple-clang compiler.cppstd=20 compiler.libcxx=libc++ compiler.version=17 build_type=Release
qt/6.8.3: options: device=None disabled_features= gui=True multiconfiguration=False opengl=desktop openssl=True qt3d=False qt5compat=False qtactiveqt=False qtcharts=False qtcoap=False qtconnectivity=False qtdatavis3d=False qtdeclarative=False qtdoc=False qtgraphs=False qtgrpc=False qthttpserver=False qtimageformats=False qtlanguageserver=False qtlocation=False qtlottie=False qtmqtt=False qtmultimedia=False qtnetworkauth=False qtopcua=False qtpositioning=False qtquick3d=False qtquick3dphysics=False qtquickeffectmaker=False qtquicktimeline=False qtremoteobjects=False qtscxml=False qtsensors=False qtserialbus=False qtserialport=False qtshadertools=False qtspeech=False qtsvg=True qttools=True qttranslations=True qtvirtualkeyboard=False qtwayland=False qtwebchannel=False qtwebengine=False qtwebsockets=False qtwebview=False shared=False widgets=True with_brotli=False with_dbus=False with_doubleconversion=True with_freetype=False with_glib=False with_gssapi=False with_harfbuzz=False with_libjpeg=False with_libpng=True with_md4c=False with_mysql=False with_odbc=False with_pcre2=True with_pq=False with_sqlite3=False with_vulkan=False with_zstd=False
qt/6.8.3: requires: openssl/3.6.Z pcre2/10.42.Z bzip2/1.0.Z double-conversion/3.3.Z libpng/1.6.Z zlib/1.3.Z opengl/system
qt/6.8.3: Building your package in /Users/gitlab/.conan2/p/b/qtc35689179027b/b
qt/6.8.3: Calling generate()
qt/6.8.3: Generators folder: /Users/gitlab/.conan2/p/b/qtc35689179027b/b/build/Release/generators
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'ZLIBConfig.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'double-conversion-config.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'module-BZip2-release-x86_64-data.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'double-conversion-config-version.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'FindOpenSSL.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'BZip2ConfigVersion.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'BZip2-release-x86_64-data.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'PNGConfigVersion.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'ZLIBConfigVersion.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'OpenSSLConfigVersion.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'PNG-release-x86_64-data.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'FindBZip2.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'module-OpenSSL-release-x86_64-data.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'conandeps_legacy.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'PCRE2-release-x86_64-data.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'PCRE2Config.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'double-conversion-release-x86_64-data.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'OpenSSLConfig.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'PCRE2ConfigVersion.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'module-PNG-release-x86_64-data.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'opengl_system-config.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'OpenSSL-release-x86_64-data.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'FindZLIB.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'opengl_system-config-version.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'module-ZLIB-release-x86_64-data.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'BZip2Config.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'ZLIB-release-x86_64-data.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'opengl_system-release-x86_64-data.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'FindPNG.cmake' file.
qt/6.8.3: WARN: replace_in_file didn't find pattern ' IMPORTED)
' in 'PNGConfig.cmake' file.
qt/6.8.3: CMakeToolchain generated: conan_toolchain.cmake
qt/6.8.3: CMakeToolchain generated: /Users/gitlab/.conan2/p/b/qtc35689179027b/b/build/Release/generators/CMakePresets.json
qt/6.8.3: CMakeToolchain generated: /Users/gitlab/.conan2/p/qt38d4ea8bc0ce1/s/src/CMakeUserPresets.json
qt/6.8.3: Generating aggregated env files
qt/6.8.3: Generated aggregated env files: ['conanbuild.sh']
qt/6.8.3: Calling build()
qt/6.8.3: Running CMake.configure()
qt/6.8.3: RUN: cmake -G "Ninja" -DCMAKE_TOOLCHAIN_FILE="/Users/gitlab/.conan2/p/b/qtc35689179027b/b/build/Release/generators/conan_toolchain.cmake" -DCMAKE_INSTALL_PREFIX="/Users/gitlab/.conan2/p/b/qtc35689179027b/p" -DCMAKE_TRY_COMPILE_CONFIGURATION="Release" -DQT_USE_VCPKG="OFF" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" -DCMAKE_BUILD_TYPE="Release" "/Users/gitlab/.conan2/p/qt38d4ea8bc0ce1/s/src"
-- Using Conan toolchain: /Users/gitlab/.conan2/p/b/qtc35689179027b/b/build/Release/generators/conan_toolchain.cmake
-- Conan toolchain: Defining architecture flag: -m64
-- Conan toolchain: Defining libcxx as C++ flags: -stdlib=libc++
-- Conan toolchain: C++ Standard 20 with extensions OFF
-- Conan toolchain: Setting BUILD_SHARED_LIBS = OFF
-- The CXX compiler identification is AppleClang 17.0.0.17000013
-- The C compiler identification is AppleClang 17.0.0.17000013
-- The ASM compiler identification is AppleClang
-- Found assembler: /Library/Developer/CommandLineTools/usr/bin/clang
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Checking dependencies of submodule 'qtbase'
-- Checking dependencies of submodule 'qtsvg'
-- Checking dependencies of submodule 'qttools'
-- Skipping optional dependency 'qtactiveqt' of 'qttools', because building 'qtactiveqt' was explicitly disabled.
-- Skipping optional dependency 'qtdeclarative' of 'qttools', because building 'qtdeclarative' was explicitly disabled.
-- Checking dependencies of submodule 'qttranslations'
-- Configuring submodule 'qtbase'
-- [QtBase] CMAKE_BUILD_TYPE was already explicitly set to: 'Release'
CMake Warning (dev) at qtbase/cmake/QtPublicSbomGenerationHelpers.cmake:73 (cmake_parse_arguments):
  The PROJECT_COMMENT keyword was followed by an empty string or no value at
  all.  Policy CMP0174 is not set, so cmake_parse_arguments() will unset the
  arg_PROJECT_COMMENT variable rather than setting it to an empty string.
Call Stack (most recent call first):
  qtbase/cmake/QtPublicSbomHelpers.cmake:217 (_qt_internal_sbom_begin_project_generate)
  qtbase/cmake/QtPublicSbomHelpers.cmake:478 (_qt_internal_sbom_begin_project)
  qtbase/cmake/QtPublicSbomHelpers.cmake:441 (_qt_internal_sbom_begin_qt_repo_project)
  qtbase/cmake/QtBuildRepoHelpers.cmake:354 (_qt_internal_sbom_auto_begin_qt_repo_project)
  qtbase/cmake/QtBaseHelpers.cmake:154 (qt_build_repo_begin)
  qtbase/CMakeLists.txt:32 (qt_internal_qtbase_build_repo)
This warning is for project developers.  Use -Wno-dev to suppress it.
-- Configuring submodule 'qtsvg'
CMake Warning (dev) at qtbase/cmake/QtPublicSbomGenerationHelpers.cmake:73 (cmake_parse_arguments):
  The PROJECT_COMMENT keyword was followed by an empty string or no value at
  all.  Policy CMP0174 is not set, so cmake_parse_arguments() will unset the
  arg_PROJECT_COMMENT variable rather than setting it to an empty string.
Call Stack (most recent call first):
  qtbase/cmake/QtPublicSbomHelpers.cmake:217 (_qt_internal_sbom_begin_project_generate)
  qtbase/cmake/QtPublicSbomHelpers.cmake:478 (_qt_internal_sbom_begin_project)
  qtbase/cmake/QtPublicSbomHelpers.cmake:441 (_qt_internal_sbom_begin_qt_repo_project)
  qtbase/cmake/QtBuildRepoHelpers.cmake:354 (_qt_internal_sbom_auto_begin_qt_repo_project)
  qtbase/cmake/QtBuildRepoHelpers.cmake:544 (qt_build_repo_begin)
  qtsvg/CMakeLists.txt:26 (qt_build_repo)
This warning is for project developers.  Use -Wno-dev to suppress it.
-- Configuring submodule 'qttools'
CMake Warning (dev) at qtbase/cmake/QtPublicSbomGenerationHelpers.cmake:73 (cmake_parse_arguments):
  The PROJECT_COMMENT keyword was followed by an empty string or no value at
  all.  Policy CMP0174 is not set, so cmake_parse_arguments() will unset the
  arg_PROJECT_COMMENT variable rather than setting it to an empty string.
Call Stack (most recent call first):
  qtbase/cmake/QtPublicSbomHelpers.cmake:217 (_qt_internal_sbom_begin_project_generate)
  qtbase/cmake/QtPublicSbomHelpers.cmake:478 (_qt_internal_sbom_begin_project)
  qtbase/cmake/QtPublicSbomHelpers.cmake:441 (_qt_internal_sbom_begin_qt_repo_project)
  qtbase/cmake/QtBuildRepoHelpers.cmake:354 (_qt_internal_sbom_auto_begin_qt_repo_project)
  qtbase/cmake/QtBuildRepoHelpers.cmake:544 (qt_build_repo_begin)
  qttools/CMakeLists.txt:27 (qt_build_repo)
This warning is for project developers.  Use -Wno-dev to suppress it.
CMake Deprecation Warning at qttools/src/assistant/qlitehtml/src/3rdparty/litehtml/src/gumbo/CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.
  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.
-- Configuring submodule 'qttranslations'
CMake Warning (dev) at qtbase/cmake/QtPublicSbomGenerationHelpers.cmake:73 (cmake_parse_arguments):
  The PROJECT_COMMENT keyword was followed by an empty string or no value at
  all.  Policy CMP0174 is not set, so cmake_parse_arguments() will unset the
  arg_PROJECT_COMMENT variable rather than setting it to an empty string.
Call Stack (most recent call first):
  qtbase/cmake/QtPublicSbomHelpers.cmake:217 (_qt_internal_sbom_begin_project_generate)
  qtbase/cmake/QtPublicSbomHelpers.cmake:478 (_qt_internal_sbom_begin_project)
  qtbase/cmake/QtPublicSbomHelpers.cmake:441 (_qt_internal_sbom_begin_qt_repo_project)
  qtbase/cmake/QtBuildRepoHelpers.cmake:354 (_qt_internal_sbom_auto_begin_qt_repo_project)
  qtbase/cmake/QtBuildRepoHelpers.cmake:544 (qt_build_repo_begin)
  qttranslations/CMakeLists.txt:21 (qt_build_repo)
This warning is for project developers.  Use -Wno-dev to suppress it.
-- Configuration summary has been written to /Users/gitlab/.conan2/p/b/qtc35689179027b/b/build/Release/config.summary
Note: Using static linking will disable the use of dynamically loaded plugins. Make sure to import all needed static plugins, or compile needed modules into the library.
WARNING: 
QDoc will not be compiled, probably because clang's C and C++ libraries could not be located.
This means that you cannot build the Qt documentation. You may need to set the CMake variables
CMAKE_PREFIX_PATH or LLVM_INSTALL_DIR to the location of your llvm installation.
Other than clang's libraries, you may need to install another package, such as clang itself, to
provide the ClangConfig.cmake file needed to detect your libraries. Once this file is in place,
the configure script may be able to detect your system-installed libraries without further
environment variables.
NOTE: You WILL also need to set the FEATURE_clang CMake variable to ON to re-try finding the
dependencies.
On macOS, you can use Homebrew's llvm package.
Run `brew install llvm` and then configure with
 `configure LLVM_INSTALL_DIR=/opt/homebrew/opt/llvm FEATURE_clang=ON` for macOS arm64
or
 `configure LLVM_INSTALL_DIR=/usr/local/opt/llvm FEATURE_clang_ON` for macOS x86_64.
On Linux, you can try installing the clang package from your distribution's package manager.
On Debian / Ubuntu run `sudo apt install libclang-dev`.
On Fedora / RHEL run `sudo dnf install clang-devel`.
On ArchLinux run `sudo pacman -S clang llvm`.
Alternatively, you can use the prebuilt binaries hosted by Qt.
These let you link LLVM/Clang libraries statically, but only support Release builds on Windows.
https://download.qt.io/development_releases/prebuilt/libclang/qt/
After installing, reconfigure with
 `/qt_src/configure FEATURE_clang=ON` for a top-level build that includes the qttools repo
or
 `/qt_build/qt-configure-module /path/to/qttools FEATURE_clang=ON` for a per-submodule build
WARNING: QDoc will not be compiled because the QmlPrivate library wasn't found.
WARNING: The Clang-based lupdate parser will not be available. Suitable LLVM and Clang C++ libraries have not been found. You will need to set the FEATURE_clangcpp CMake variable to ON to re-evaluate this check.
-- 
Qt is now configured for building. Just run 'cmake --build . --parallel'
Once everything is built, you must run 'cmake --install .'
Qt will be installed into '/Users/gitlab/.conan2/p/b/qtc35689179027b/p'
To configure and build other Qt modules, you can use the following convenience script:
        /Users/gitlab/.conan2/p/b/qtc35689179027b/p/bin/qt-configure-module
If reconfiguration fails for some reason, try removing 'CMakeCache.txt' from the build directory
Alternatively, you can add the --fresh flag to your CMake flags.
-- Configuring done (86.4s)
-- Generating done (3.8s)
-- Build files have been written to: /Users/gitlab/.conan2/p/b/qtc35689179027b/b/build/Release
qt/6.8.3: Running CMake.build()
qt/6.8.3: RUN: cmake --build "/Users/gitlab/.conan2/p/b/qtc35689179027b/b/build/Release" -- -j4
[...]

qt/6.8.3: Created package revision d63a9484ecfc2d5cd4fb5992324c1060
qt/6.8.3: Package '92b716dd6a5affd19225a692e1b7945296daba4f' created
qt/6.8.3: Full package reference: qt/6.8.3#f6ec3d69031717def8d69ad53bcf8558:92b716dd6a5affd19225a692e1b7945296daba4f#d63a9484ecfc2d5cd4fb5992324c1060
qt/6.8.3: Package folder /Users/gitlab/.conan2/p/b/qtc35689179027b/p
WARN: deprecated: Usage of deprecated Conan 1.X features that will be removed in Conan 2.X:
WARN: deprecated:     'cpp_info.names' used in: bzip2/1.0.8, protobuf/3.21.12, boost/1.89.0, zlib/1.3.1, sqlite_orm/1.9.1, inja/3.5.0, libarchive/3.8.1, libpng/1.6.50
WARN: deprecated:     'cpp_info.build_modules' used in: bzip2/1.0.8, protobuf/3.21.12
WARN: deprecated:     'env_info' used in: bzip2/1.0.8, protobuf/3.21.12, boost/1.89.0
WARN: deprecated:     'cpp_info.filenames' used in: protobuf/3.21.12, boost/1.89.0, sqlite_orm/1.9.1, opengl/system, inja/3.5.0
WARN: deprecated:     'user_info' used in: boost/1.89.0
======== Finalizing install (deploy, generators) ========
conanfile.py (pep/None): Calling generate()
conanfile.py (pep/None): Generators folder: /Users/gitlab/builds/SF_xY11J8/0/pep/core/build/generators
conanfile.py (pep/None): CMakeDeps necessary find_package() and targets for your CMakeLists.txt
    find_package(LibArchive)
    find_package(Boost)
    find_package(civetweb)
    find_package(inja)
    find_package(nlohmann_json)
    find_package(protobuf)
    find_package(Qt6)
    find_package(OpenSSL)
    find_package(SqliteOrm)
    find_package(xxHash)
    target_link_libraries(... LibArchive::LibArchive boost::boost civetweb::civetweb-cpp pantor::inja nlohmann_json::nlohmann_json protobuf::protobuf qt::qt openssl::openssl sqlite_orm::sqlite_orm xxHash::xxhash)
conanfile.py (pep/None): CMakeToolchain generated: conan_toolchain.cmake
conanfile.py (pep/None): CMakeToolchain: Preset 'conan-release' added to CMakePresets.json.
    (cmake>=3.23) cmake --preset conan-release
    (cmake<3.23) cmake <path> -G Ninja -DCMAKE_TOOLCHAIN_FILE=generators/conan_toolchain.cmake  -DCMAKE_BUILD_TYPE=Release -DWITH_ASSESSOR=ON -DWITH_LOGON=ON -DWITH_SERVERS=OFF -DWITH_CASTOR=OFF -DWITH_TESTS=OFF -DWITH_BENCHMARK=OFF -DWITH_UNWINDER=OFF -DCMAKE_POLICY_DEFAULT_CMP0091=NEW
conanfile.py (pep/None): CMakeToolchain generated: /Users/gitlab/builds/SF_xY11J8/0/pep/core/build/generators/CMakePresets.json
conanfile.py (pep/None): CMakeToolchain generated: /Users/gitlab/builds/SF_xY11J8/0/pep/core/CMakeUserPresets.json
conanfile.py (pep/None): Generating aggregated env files
conanfile.py (pep/None): Generated aggregated env files: ['conanbuild.sh', 'conanrun.sh']
Install finished successfully
```

Now building our app:
```
Configuring CMake project.
-- Using Conan toolchain: /Users/gitlab/builds/SF_xY11J8/0/pep/core/build/generators/conan_toolchain.cmake
-- Conan toolchain: Defining architecture flag: -m64
-- Conan toolchain: Defining libcxx as C++ flags: -stdlib=libc++
-- Conan toolchain: C++ Standard 20 with extensions OFF
-- The C compiler identification is AppleClang 17.0.0.17000013
-- The CXX compiler identification is AppleClang 17.0.0.17000013
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
[...]
-- Conan: Component target declared 'Qt6::Platform'
-- Conan: Component target declared 'qt::QCocoaIntegrationPlugin'
-- Conan: Component target declared 'Qt6::LinguistTools'
-- Conan: Component target declared 'Qt6::Core'
-- Conan: Component target declared 'Qt6::Gui'
-- Conan: Component target declared 'Qt6::Network'
-- Conan: Component target declared 'Qt6::Sql'
-- Conan: Component target declared 'Qt6::Test'
-- Conan: Component target declared 'Qt6::Concurrent'
-- Conan: Component target declared 'Qt6::Xml'
-- Conan: Component target declared 'Qt6::QSvgIconPlugin'
-- Conan: Component target declared 'Qt6::QSvgPlugin'
-- Conan: Component target declared 'Qt6::QCocoaIntegrationPlugin'
-- Conan: Component target declared 'Qt6::QGifPlugin'
-- Conan: Component target declared 'Qt6::QIcoPlugin'
-- Conan: Component target declared 'Qt6::Widgets'
-- Conan: Component target declared 'Qt6::OpenGL'
-- Conan: Component target declared 'Qt6::Svg'
-- Conan: Component target declared 'Qt6::PrintSupport'
-- Conan: Component target declared 'Qt6::OpenGLWidgets'
-- Conan: Component target declared 'Qt6::UiPlugin'
-- Conan: Component target declared 'Qt6::Help'
-- Conan: Component target declared 'Qt6::SvgWidgets'
-- Conan: Component target declared 'Qt6::UiTools'
-- Conan: Component target declared 'Qt6::Designer'
-- Conan: Target declared 'qt::qt'
-- Conan: Component target declared 'OpenSSL::Crypto'
-- Conan: Component target declared 'OpenSSL::SSL'
-- Conan: Target declared 'openssl::openssl'
-- Conan: Target declared 'ZLIB::ZLIB'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/opens9af3c6f9e5ecc/p/lib/cmake/conan-official-openssl-variables.cmake'
-- Conan: Component target declared 'PCRE2::8BIT'
-- Conan: Component target declared 'PCRE2::16BIT'
-- Conan: Component target declared 'PCRE2::32BIT'
-- Conan: Component target declared 'PCRE2::POSIX'
-- Conan: Target declared 'pcre2::pcre2'
-- Conan: Target declared 'BZip2::BZip2'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/bzip2cfbd4df2bf4c4/p/lib/cmake/conan-official-bzip2-variables.cmake'
-- Conan: Target declared 'double-conversion::double-conversion'
-- Conan: Target declared 'PNG::PNG'
-- Conan: Target declared 'opengl::opengl'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6Core/conan_qt_executables_variables.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6Core/conan_qt_qt6_coreprivate.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6Core/Qt6CoreMacros.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6Core/Qt6CoreConfigExtras.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/Qt6ConfigExtras.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicExternalProjectHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicCMakeVersionHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicCMakeHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomPurlHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicGitHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomAttributionHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicDependencyHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicAppleHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomFileHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicTargetHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomSystemDepHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicFinalizerHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomOpsHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomCpeHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicTestHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomQtEntityHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomLicenseHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicWalkLibsHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicToolHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomGenerationHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicPluginHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicFindPackageHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicWasmToolchainHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomPythonHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomDepHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6Gui/conan_qt_qt6_guiprivate.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6Widgets/conan_qt_qt6_widgetsprivate.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6Widgets/Qt6WidgetsMacros.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6LinguistTools/Qt6LinguistToolsMacros.cmake'
-- Conan: Component target declared 'Qt6::Platform'
-- Conan: Component target declared 'qt::QCocoaIntegrationPlugin'
-- Conan: Component target declared 'Qt6::LinguistTools'
-- Conan: Component target declared 'Qt6::Core'
-- Conan: Component target declared 'Qt6::Gui'
-- Conan: Component target declared 'Qt6::Network'
-- Conan: Component target declared 'Qt6::Sql'
-- Conan: Component target declared 'Qt6::Test'
-- Conan: Component target declared 'Qt6::Concurrent'
-- Conan: Component target declared 'Qt6::Xml'
-- Conan: Component target declared 'Qt6::QSvgIconPlugin'
-- Conan: Component target declared 'Qt6::QSvgPlugin'
-- Conan: Component target declared 'Qt6::QCocoaIntegrationPlugin'
-- Conan: Component target declared 'Qt6::QGifPlugin'
-- Conan: Component target declared 'Qt6::QIcoPlugin'
-- Conan: Component target declared 'Qt6::Widgets'
-- Conan: Component target declared 'Qt6::OpenGL'
-- Conan: Component target declared 'Qt6::Svg'
-- Conan: Component target declared 'Qt6::PrintSupport'
-- Conan: Component target declared 'Qt6::OpenGLWidgets'
-- Conan: Component target declared 'Qt6::UiPlugin'
-- Conan: Component target declared 'Qt6::Help'
-- Conan: Component target declared 'Qt6::SvgWidgets'
-- Conan: Component target declared 'Qt6::UiTools'
-- Conan: Component target declared 'Qt6::Designer'
-- Conan: Target declared 'qt::qt'
-- Conan: Component target declared 'OpenSSL::Crypto'
-- Conan: Component target declared 'OpenSSL::SSL'
-- Conan: Target declared 'openssl::openssl'
-- Conan: Target declared 'ZLIB::ZLIB'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/opens9af3c6f9e5ecc/p/lib/cmake/conan-official-openssl-variables.cmake'
-- Conan: Component target declared 'PCRE2::8BIT'
-- Conan: Component target declared 'PCRE2::16BIT'
-- Conan: Component target declared 'PCRE2::32BIT'
-- Conan: Component target declared 'PCRE2::POSIX'
-- Conan: Target declared 'pcre2::pcre2'
-- Conan: Target declared 'BZip2::BZip2'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/bzip2cfbd4df2bf4c4/p/lib/cmake/conan-official-bzip2-variables.cmake'
-- Conan: Target declared 'double-conversion::double-conversion'
-- Conan: Target declared 'PNG::PNG'
-- Conan: Target declared 'opengl::opengl'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6Core/conan_qt_executables_variables.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6Core/conan_qt_qt6_coreprivate.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6Core/Qt6CoreMacros.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6Core/Qt6CoreConfigExtras.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/Qt6ConfigExtras.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicExternalProjectHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicCMakeVersionHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicCMakeHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomPurlHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicGitHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomAttributionHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicDependencyHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicAppleHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomFileHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicTargetHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomSystemDepHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicFinalizerHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomOpsHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomCpeHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicTestHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomQtEntityHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomLicenseHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicWalkLibsHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicToolHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomGenerationHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicPluginHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicFindPackageHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicWasmToolchainHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomPythonHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6/QtPublicSbomDepHelpers.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6Gui/conan_qt_qt6_guiprivate.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6Widgets/conan_qt_qt6_widgetsprivate.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6Widgets/Qt6WidgetsMacros.cmake'
-- Conan: Including build module from '/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/cmake/Qt6LinguistTools/Qt6LinguistToolsMacros.cmake'
[...]
-- Configuring done (29.4s)
-- Generating done (1.6s)
-- Build files have been written to: /Users/gitlab/builds/SF_xY11J8/0/pep/core/build
```

```
[44/44] Linking CXX executable cpp/pep/assessor/pepAssessor.app/Contents/MacOS/pepAssessor
FAILED: [code=1] cpp/pep/assessor/pepAssessor.app/Contents/MacOS/pepAssessor 
: && /usr/bin/clang++ -m64 -stdlib=libc++ -fvisibility-inlines-hidden -fvisibility=hidden -DBOOST_LOG_USE_NATIVE_SYSLOG -DBOOST_PROCESS_USE_STD_FS -Werror -DBUILD_PROJECT_PATH=\"pep/core\" -DBUILD_REF=\"macos-qt-fix\" -DBUILD_REV=\"2cf862c85601dc011b06c6b6300817836cd5c265\" -DBUILD_MAJOR_VERSION=1 -DBUILD_MINOR_VERSION=6 -DBUILD_PIPELINE_ID=59199 -DBUILD_JOB_ID=695677 -DBUILD_TARGET=\"Darwin\" -O3 -DNDEBUG -arch x86_64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -mmacosx-version-min=13.3 -Wl,-search_paths_first -Wl,-headerpad_max_install_names -m64 -Wl,-fatal_warnings,-no_warn_duplicate_libraries /Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/objects-Release/PrintSupport_resources_1/.qt/rcc/qrc_qprintdialog_init.cpp.o /Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/objects-Release/PrintSupport_resources_2/.qt/rcc/qrc_qprintdialog1_init.cpp.o /Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/objects-Release/Gui_resources_2/.qt/rcc/qrc_gui_shaders_init.cpp.o /Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/objects-Release/Gui_resources_1/.qt/rcc/qrc_qpdf_init.cpp.o /Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/objects-Release/Widgets_resources_3/.qt/rcc/qrc_qstyle_fusion_init.cpp.o /Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/objects-Release/Widgets_resources_4/.qt/rcc/qrc_qmessagebox_init.cpp.o /Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/objects-Release/Widgets_resources_2/.qt/rcc/qrc_qstyle1_init.cpp.o /Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/objects-Release/Widgets_resources_1/.qt/rcc/qrc_qstyle_init.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/pepAssessor_autogen/mocs_compilation.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/Async.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/Branding.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/Branding.PropertySerializer.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/ExportableItem.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/InputValidationTooltip.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/assessorwidget.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/dateeditor.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/datetimeeditor.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/devicehistorywidget.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/devicewidget.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/enrollmentwidget.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/exportwidget.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/Installer.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/loginwidget.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/main.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/mainwindow.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/notconnectedwidget.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/participanteditor.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/participantselector.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/participantwidget.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/QDate.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/visitwidget.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/FlowLayout.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/ButtonBar.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/UserRole.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/SparkleUpdater.mm.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/qrc_resources.cpp.o cpp/pep/assessor/CMakeFiles/pepAssessor.dir/.qt/rcc/qrc_pepAssessor_translations.cpp.o -o cpp/pep/assessor/pepAssessor.app/Contents/MacOS/pepAssessor -F/usr/local/Caskroom/sparkle/2.8.0  -L/Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib   -L/Users/gitlab/.conan2/p/b/opens9af3c6f9e5ecc/p/lib   -L/Users/gitlab/.conan2/p/b/zlib39c5f7de62f93/p/lib   -L/Users/gitlab/.conan2/p/b/xxhas573a8f75cdb90/p/lib   -L/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib   -L/Users/gitlab/.conan2/p/b/pcre2604e686d1502d/p/lib   -L/Users/gitlab/.conan2/p/b/bzip2cfbd4df2bf4c4/p/lib   -L/Users/gitlab/.conan2/p/b/doubl484aa52d172d7/p/lib   -L/Users/gitlab/.conan2/p/b/libpnf35af779078cd/p/lib   -L/Users/gitlab/.conan2/p/b/protoede04dfc41f01/p/lib   -L/Users/gitlab/.conan2/p/b/civetca100b07453c0/p/lib -Wl,-rpath,/Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib -Wl,-rpath,/Users/gitlab/.conan2/p/b/opens9af3c6f9e5ecc/p/lib -Wl,-rpath,/Users/gitlab/.conan2/p/b/zlib39c5f7de62f93/p/lib -Wl,-rpath,/Users/gitlab/.conan2/p/b/xxhas573a8f75cdb90/p/lib -Wl,-rpath,/Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib -Wl,-rpath,/Users/gitlab/.conan2/p/b/pcre2604e686d1502d/p/lib -Wl,-rpath,/Users/gitlab/.conan2/p/b/bzip2cfbd4df2bf4c4/p/lib -Wl,-rpath,/Users/gitlab/.conan2/p/b/doubl484aa52d172d7/p/lib -Wl,-rpath,/Users/gitlab/.conan2/p/b/libpnf35af779078cd/p/lib -Wl,-rpath,/Users/gitlab/.conan2/p/b/protoede04dfc41f01/p/lib -Wl,-rpath,/Users/gitlab/.conan2/p/b/civetca100b07453c0/p/lib -Wl,-rpath,/usr/local/Caskroom/sparkle/2.8.0  cpp/pep/gui/libpepGuilib.a  cpp/pep/oauth-client/libpepOAuthClientlib.a  cpp/pep/client/libpepClientlib.a  cpp/pep/application/libpepApplicationlib.a  -framework Sparkle  /Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/libQt6PrintSupport.a  -lcups  /Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/libQt6Svg.a  /Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/libQt6Widgets.a  /Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/libQt6Gui.a  /Users/gitlab/.conan2/p/b/qtc35689179027b/p/lib/libQt6Core.a  -framework IOKit  -framework UniformTypeIdentifiers  -framework ApplicationServices  -framework CoreServices  -framework Security  -framework DiskArbitration  /Users/gitlab/.conan2/p/b/pcre2604e686d1502d/p/lib/libpcre2-posix.a  /Users/gitlab/.conan2/p/b/pcre2604e686d1502d/p/lib/libpcre2-32.a  /Users/gitlab/.conan2/p/b/pcre2604e686d1502d/p/lib/libpcre2-16.a  /Users/gitlab/.conan2/p/b/pcre2604e686d1502d/p/lib/libpcre2-8.a  /Users/gitlab/.conan2/p/b/bzip2cfbd4df2bf4c4/p/lib/libbz2.a  /Users/gitlab/.conan2/p/b/doubl484aa52d172d7/p/lib/libdouble-conversion.a  /Users/gitlab/.conan2/p/b/opens9af3c6f9e5ecc/p/lib/libssl.a  /Users/gitlab/.conan2/p/b/opens9af3c6f9e5ecc/p/lib/libcrypto.a  -framework CoreFoundation  -framework CoreGraphics  -framework CoreText  -framework Foundation  -framework ImageIO  -framework AppKit  -framework Carbon  -framework Metal  /Users/gitlab/.conan2/p/b/libpnf35af779078cd/p/lib/libpng.a  /Users/gitlab/.conan2/p/b/zlib39c5f7de62f93/p/lib/libz.a  -framework OpenGL  cpp/pep/httpserver/libpepHttpserverlib.a  /Users/gitlab/.conan2/p/b/civetca100b07453c0/p/lib/libcivetweb.a  -framework Cocoa  /Users/gitlab/.conan2/p/b/opens9af3c6f9e5ecc/p/lib/libssl.a  /Users/gitlab/.conan2/p/b/opens9af3c6f9e5ecc/p/lib/libcrypto.a  /Users/gitlab/.conan2/p/b/zlib39c5f7de62f93/p/lib/libz.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_process.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_filesystem.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_atomic.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_context.a  cpp/pep/authserver/libpepAuthserverApilib.a  cpp/pep/core-client/libpepCoreClientlib.a  cpp/pep/accessmanager/libpepAccessManagerApilib.a  cpp/pep/structure/libpepStructurelib.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_random.a  cpp/pep/storagefacility/libpepStorageFacilityApilib.a  cpp/pep/transcryptor/libpepTranscryptorApilib.a  cpp/pep/morphing/libpepMorphinglib.a  cpp/pep/auth/libpepAuthlib.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_url.a  cpp/pep/ticketing/libpepTicketinglib.a  cpp/pep/content/libpepContentlib.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_date_time.a  cpp/pep/keyserver/libpepKeyServerApilib.a  cpp/pep/registrationserver/libpepRegistrationServerApilib.a  cpp/pep/server/libpepServerApilib.a  cpp/pep/messaging/libpepMessaginglib.a  cpp/pep/networking/libpepNetworkinglib.a  cpp/pep/async/libpepAsynclib.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_url.a  /Users/gitlab/.conan2/p/b/opens9af3c6f9e5ecc/p/lib/libssl.a  /Users/gitlab/.conan2/p/b/opens9af3c6f9e5ecc/p/lib/libcrypto.a  /Users/gitlab/.conan2/p/b/zlib39c5f7de62f93/p/lib/libz.a  cpp/pep/rsk-pep/libpepRskPeplib.a  cpp/pep/rsk/libpepRsklib.a  cpp/pep/elgamal/libpepElgamallib.a  cpp/pep/crypto/libpepCryptolib.a  cpp/pep/serialization/libpepSerializationlib.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_date_time.a  /Users/gitlab/.conan2/p/b/opens9af3c6f9e5ecc/p/lib/libcrypto.a  /Users/gitlab/.conan2/p/b/zlib39c5f7de62f93/p/lib/libz.a  cpp/ext/panda/libpanda.a  cpp/pep/versioning/libpepVersioninglib.a  cpp/pep/utils/libpepUtilslib.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_log_setup.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_log.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_filesystem.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_regex.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_thread.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_exception.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_atomic.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_date_time.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_chrono.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_container.a  /Users/gitlab/.conan2/p/b/opens9af3c6f9e5ecc/p/lib/libcrypto.a  /Users/gitlab/.conan2/p/b/zlib39c5f7de62f93/p/lib/libz.a  /Users/gitlab/.conan2/p/b/xxhas573a8f75cdb90/p/lib/libxxhash.a  /Users/gitlab/.conan2/p/b/protoede04dfc41f01/p/lib/libprotobuf.a  /Users/gitlab/.conan2/p/b/zlib39c5f7de62f93/p/lib/libz.a  /Users/gitlab/.conan2/p/b/boost67dc7ee91b4c6/p/lib/libboost_date_time.a

Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$_CAMetalLayer", referenced from:
      _OBJC_CLASS_$_QMetalLayer in libQt6Gui.a[197](qmetallayer.mm.o)
  "_OBJC_METACLASS_$_CAMetalLayer", referenced from:
      _OBJC_METACLASS_$_QMetalLayer in libQt6Gui.a[197](qmetallayer.mm.o)
ld: symbol(s) not found for architecture x86_64
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
ninja: build stopped: subcommand failed.
```

</details>

--- a/recipes/qt/6.x.x/conanfile.py
+++ b/recipes/qt/6.x.x/conanfile.py
@@ -1209,10 +1209,10 @@ class QtConan(ConanFile):
             elif is_apple_os(self):
                 # https://github.com/qt/qtbase/blob/v6.6.1/src/gui/CMakeLists.txt#L388-L394
                 self.cpp_info.components["qtGui"].frameworks = ["CoreFoundation", "CoreGraphics", "CoreText", "Foundation", "ImageIO"]
-                # https://github.com/qt/qtbase/blob/6.8.3/src/gui/configure.cmake#L841-L844
+                # https://github.com/qt/qtbase/blob/6.8.0/src/gui/configure.cmake#L834-L837
                 has_metal = "metal" not in disabled_features and self.settings.os in ["Macos", "iOS", "visionOS"]
-                if has_metal:
-                    # https://github.com/qt/qtbase/blob/6.8.3/src/gui/CMakeLists.txt#L434-L439
+                if Version(self.version) >= "6.8.0" and has_metal:
+                    # https://github.com/qt/qtbase/blob/6.8.0/src/gui/CMakeLists.txt#L432-L437
                     self.cpp_info.components["qtGui"].frameworks.append("QuartzCore")
                 if self.settings.os == "Macos":
                     # https://github.com/qt/qtbase/blob/v6.6.1/src/gui/CMakeLists.txt#L362-L370

